### ISSUE TITLE
Enable reflow/zoom for pre blocks

### ISSUE THREAD

**Issue by alastc** [w3c/tr-design#168]
For code samples in the WCAG technique documents, the pre elements cause horizontal scrolling for readers zooming in ([reported to the WCAG repo here](https://github.com/w3c/wcag/issues/423)). 

I think that adding `white-space: pre-wrap;` to pre (after line 499 of base.css) would resolve this.

For people with very little horizontal space available the lines would wrap, but if you do have a larger display it would behave as it does now.

It's a simple thing so I was going to do a pull request, but I'm not sure I have permissions for that, and this repo seems so clean of branches I didn't want to add another one!

Is there some impact I haven't thought about, or could this be added?

**Comment by fantasai** [comment]
I didn't want to add pre-wrap to the default W3C-wide style sheet because for some uses of PRE, line-wrapping would be misleading. For example, some CSS specifications use PRE text to show how text is expected to wrap under certain conditions--introducing soft line breaks would break the examples! 
Another downside of pre-wrap is that the wrapped lines don't match the previous line's indentation level, which can be confusing. Scrolling is annoying, but it does avoid these problems.

There's no `white-space` rule for pre in base.css, so you should be able to add it to your spec's own style sheets without any problem. At least within your own specs, you can know whether pre-wrap is appropriate.

Let me know if that makes sense or if you have some other point to consider for base.css. :)

**Comment by alastc** [comment]
Ah, fair enough, we'll do that locally.
